.github/workflows/generate-changelog.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
```

Comment:
[q] just want to confirm we added `svc-apix-bot` within our branch protection rules

Reply:
yes I added an exemption for svc-api-bot to bypass the branch protection rule